### ISSUE TITLE
Atropos - Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by arnaudbore** [ANTsX/ANTs#1756]
### Operating system and version

Ubuntu 22.04

### CPU architecture

x86_64 (PC, Intel Mac, other Intel/AMD)

### ANTs code version

2.5.0.post11-gdee5e1a

### ANTs installation type

Compiled from source

### Summary of the problem

When trying to run Atropos I get a Segmentation fault (core dumped).
Seems to work with v2.3.5.post76-g28dd25c

Not working with: v2.4.3-gcaa60eb or v2.5.0.post11-gdee5e1a

### Commands to reproduce the problem.
```
Atropos -a [122317_b0b1000_FA.nii.gz] -a [122317_mean_b1000_n4.nii.gz] -x 122317_mask.nii.gz -i PriorProbabilityImages[3, tissue_prob_122317_%02d.nii.gz,0.1,0.0001] -m [0.3, 2x2x2]  -s 1x2 -s 1x3 -s 2x3  --use-partial-volume-likelihoods false  -o [tissue_prob_122317_out_seg.nii.gz,tissue_prob_122317_out_prob_%02d.nii.gz] -k HistogramParzenWindows[1.0,32]
```
### Output of the command with verbose output.

Nothing in my_log.txt

### Data to reproduce the problem

Here are the data used to reproduce the error: 
https://box.criugm.qc.ca/f/3be6c828cc334030bc42/?dl=1

**Comment by cookpa** [comment]
Thanks, this works for me with the latest ANTs, so I think what you're seeing is the same thing fixed in #1663

**Comment by arnaudbore** [comment]
It works with the latest version thank you.
